Editor/UI/Wizard/FirstTime/AllAboutPage.cs

```diff
@@ -12,7 +13,23 @@ public AllAboutPage() : base("Step 4: Help & Documentation")
 
         public override void Draw(Rect window)
         {
+            GUILayout.BeginArea(window);
+                GUILayout.Label("Hit Play to Preview", CreatorEditorStyles.Title);
+                GUILayout.Label("Have a look at How-Tos and an in-depth Webinar for further information.", CreatorEditorStyles.Paragraph);
+                GUILayout.Label("How-To's", CreatorEditorStyles.Header);
 
+                CreatorLayout.DrawLink("How to build your VR Training application", "https://developers.innoactive.de/documentation/creator/latest/articles/getting-started/designer.html");
+                CreatorLayout.DrawLink("How to extend the Creator using a training template", "https://developers.innoactive.de/documentation/creator/latest/articles/developer/01-introduction.html");
+
+                GUILayout.Label("Need Help?", CreatorEditorStyles.Header);
+
+                CreatorLayout.DrawLink("In-depth webinar on how the Creator works", () => {});
```

Comment:
We do not have a webinar yet, right? So maybe comment this so it isn't shown

Reply:
added the link :)